
async function excluirMonstro(idMonstro) {
      const response = await fetch(`/user/monster/deletar/${idMonstro}`, {
        method: 'DELETE',
      })
      .catch(error => {
        throw new Error("NÃ£o excluiu")
      })

      window.location.reload();
}